src/libs/OptionsListUtils.js

```diff
@@ -402,9 +402,39 @@ function getSidebarOptions(reports, personalDetails, draftComments, activeReport
     });
 }
 
+/**
+ * Helper method that returns the text to be used for the header's message and title (if any)
+ *
+ * @param {Boolean} hasSelectableOptions
+ * @param {Boolean} hasUserToInvite
+ * @param {Boolean} [maxParticipantsReached]
+ * @return {String}
+ */
+function getHeaderTitleAndMessage(hasSelectableOptions, hasUserToInvite, maxParticipantsReached = false) {
+    if (maxParticipantsReached) {
+        return {
+            headerTitle: '',
+            headerMessage: CONST.MESSAGES.MAXIMUM_PARTICIPANTS_REACHED,
+        };
+    }
+
+    if (!hasSelectableOptions && !hasUserToInvite) {
+        return {
+            headerTitle: '',
+            headerMessage: CONST.MESSAGES.NO_CONTACTS_FOUND,
+        };
+    }
+
+    return {
+        headerTitle: '',
```

Comment:
Why do we return an empty string for headerTitle every time?

Reply:
I'm guessing it was a required prop at some point. Tbh didn't look too carefully at whether it was still being used and just copy pasta'd this. Can definitely be removed if we are not using it anywhere...

Reply:
Yeah I don't think it's used at all...